docs/machine-learning/tutorials/sentiment-analysis.md

```diff
@@ -113,29 +134,41 @@ Remove the existing class definition and add the following code, which has two c
 
 `SentimentData` is the input dataset class and has a `float` (`Sentiment`) that has a value for sentiment of either positive or negative, and a string for the comment (`SentimentText`). Both fields have `Column` attributes attached to them. This attribute describes the order of each field in the data file, and which is the `Label` field. `SentimentPrediction` is the class used for prediction after the model has been trained. It has a single boolean (`Sentiment`) and a `PredictedLabel` `ColumnName` attribute. The `Label` is used to create and train the model, and it's also used with a second dataset to evaluate the model. The `PredictedLabel` is used during prediction and evaluation. For evaluation, an input with training data, the predicted values, and the model are used.
 
-In the *Program.cs* file, replace the `Console.WriteLine("Hello World!")` line with the following code in the `Main` method:
+In the *Program.cs* file, change the `Main` method signature by replacing `void` with `async Task`, as in the following example:
+
+```csharp
+static async Task Main(string[] args)
+```
+
+You add `async` to `Main` with a <xref:System.Threading.Tasks.Task> return type because you're saving the model to a zip file later, and the program needs to wait until that external task completes.
+
+> [!NOTE]
+>An *async main* method enables you to use `await` in your `Main` method. You can read more about the details in the
+[async main](../../../docs/csharp/programming-guide/main-and-command-args/index.md) topic in the C# programming guide.
```

Comment:
Boilerplate sentence for linking to other topics is For more information about x, see... or For more information, see...